.github/workflows/ci.yaml

```
@@ -0,0 +1,28 @@
+name: CI
+on:
+  push:
+    branches: [main]
+  pull_request:
+jobs:
+  lint:
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Any reason for using a no-longer-supported version of node JS? I would recommend the LTS version (18).

```suggestion
        node-version: [18.x]
```